### PR TITLE
Insert an option to not validate the certificate in the client using SSL

### PR DESCRIPTION
**Pull request by FunCyRanger**
My intension was to connect my Hisense-TV to ioBroker.
Therefore I used the following knowledge bases:
https://github.com/mqttjs/MQTT.js
https://github.com/Krazy998/mqtt-hisensetv/blob/master/README.md

The brake through was to disable the validation of the certificate of the TV. This needs just an additional option in the MQTT-client, which I have added.

### CODE REVIEW

File: .github/workflows/test-and-release.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x]
+        node-version: [12.x, 15.x]
```
Comment (Apollon77) [inline .github/workflows/test-and-release.yml:28]: why you did that change? Should be unneeded here

**Review by Apollon77 — CHANGES_REQUESTED** [review]
You also need to update  io.package.json and add rejectUnauthorized with the default value in the native section.

I would then take care of the rest for an update after merge

**Comment by Apollon77** [comment]
Thank you, please see my review comments and question

File: .github/workflows/test-and-release.yml (hunk 1 above)
Comment (FunCyRanger) [reply .github/workflows/test-and-release.yml:28]: You are right. Just go over it.

Comment (Apollon77) [reply .github/workflows/test-and-release.yml:28]: could you please remove it again?  :-)

**Comment by Apollon77** [comment]
@FunCyRanger and lease see the "Normal" comment on the io-package change

File: .github/workflows/test-and-release.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x]
```
Comment (FunCyRanger) [inline .github/workflows/test-and-release.yml]: sorry, it's my first pull request and review. node-version 12.x is enough. how can I delete my change?

**Review by FunCyRanger — COMMENTED** [review]
hope I have deleted my change in the test-and-release.yml regarding node

**Pull request merged**